src/tokens/token.ts

```diff
@@ -149,6 +161,11 @@ export const tokenDataByNetwork: Record<
     yvCurve_stETH: "0xdCD90C7f6324cfa40d7169ef80b12031770B4325",
     yvCurve_FRAX: "0xB4AdA607B9d6b2c9Ee07A275e9616B84AC560139",
 
+    // BALANCER TOKENS
+    "50OHM-50DAI": "0x76FCf0e8C7Ff37A47a799FA2cd4c13cDe0D981C9",
+    "50OHM-50WETH": "0xD1eC5e215E8148D76F4460e4097FD3d5ae0A3558",
+    "OHM-wstETH": "0xd4f79CA0Ac83192693bce4699d0c10C66Aa6Cf0F",
```

Comment:
How to check that this is the correct address?

Reply:
Please use etherscan, it should have the token tracker

Reply:
checked in etherscan, looks ok